raiden/tests/unit/test_startup.py

```diff
@@ -0,0 +1,156 @@
+from copy import deepcopy
+from typing import Any, Dict
+
+import pytest
+
+from raiden.app import App
+from raiden.constants import Environment
+from raiden.ui.startup import setup_contracts_or_exit, setup_environment, setup_network_id_or_exit
+from raiden_contracts.constants import (
+    CONTRACT_ENDPOINT_REGISTRY,
+    CONTRACT_SECRET_REGISTRY,
+    CONTRACT_SERVICE_REGISTRY,
+    CONTRACT_TOKEN_NETWORK_REGISTRY,
+    CONTRACT_USER_DEPOSIT,
+)
+
+
+class MockWeb3Version():
+
+    def __init__(self, netid):
+        self.network = netid
+
+
+class MockWeb3():
+
+    def __init__(self, netid):
+        self.version = MockWeb3Version(netid)
+
+
+def test_setup_network_id():
+    config = {}
+
+    # Normal test
+    netid, known = setup_network_id_or_exit(config, 68, MockWeb3(68))
```

Comment:
why call this `netid` and not `chainid`

Reply:
Two reasons:

1. This is how it's already defined in our startup code that I was abstracting
2. `chain_id` is deprecated as a name and it should always now be called network_id. This has been standarized recently in [EIP 1474](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1474.md#description-4). Even web3.py is giving up warnings for it now: https://github.com/raiden-network/raiden/issues/3727

Reply:
Then you should change the key in ´config['chain_id']` as well

Reply:
I want to keep the changes in each PR I make as minimal as possible and not touch the config keys if I don't have to. Will check to see how many places it's used at, maybe it's not much.